engines/sci/graphics/text32.h

```diff
@@ -362,15 +367,15 @@ class GfxText32 {
 
 	/**
 	 * The size of the x-dimension of the coordinate system
-	 * used by the text renderer.
+	 * used by the text renderer. NB: static
```

Comment:
However, it would be good to add a comment about the fact that these are static because they were shared in the original code. I think this is reflected in the commit log, but not in the actual code docs?